lib/src/components/joystick/joystick_events.dart

```diff
@@ -7,7 +7,7 @@ enum JoystickMoveDirectional {
   MOVE_DOWN_RIGHT,
   MOVE_DOWN_LEFT,
   MOVE_LEFT,
-  IDLE
+  IDLE,
```

Comment:
Note to reviewers: This line broke on my editor, but not on the linter, could someone open this file to confirm if this is a bug on my plugin?

Reply:
working fine for me. and I believe dart supports trailing commas on enums so should work?

Reply:
Nice. I imagined it was a bug on my plugin. But it vanished eventually, thanks for confirming